src/ceph-volume/plugin/zfs/setup.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""The setup script."""
+
+from setuptools import setup, find_packages
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('HISTORY.rst') as history_file:
+    history = history_file.read()
+
+requirements = [ ]
+
+setup_requirements = [ ]
+
+test_requirements = [ ]
+
+setup(
+    author="Willem Jan Withagen",
+    author_email='wjw@digiware.nl',
+    classifiers=[
+        'Development Status :: 2 - Pre-Alpha',
+        'Intended Audience :: Developers',
```

Comment:
i don't think ceph-volume or ceph-volume-zfs' intended audience is developer. it's supposed to be used by "Information Technology" and "System Administrators", see https://wiki.python.org/moin/RevisedPyPiCategories